_includes/js/prettier_changelog.js

```diff
@@ -0,0 +1,73 @@
+var headings = document.querySelectorAll('h2,h3');
+/* Store last level 2 heading */
+var lastHeadingTwo = '';
+/* Regex for ISO-like date strings */
+var dateRE = new RegExp('\\d{2,4}\\-\\d{1,2}\\-\\d{1,2}');
+
+function getHref (element)
+{
+  /*
+   * get the href value without the # and return as a string.
+   */
+  return element.firstChild.attributes.href.value.match(/[^\#]+/).toString();
+}
+
+function setHref (element, newValue)
+{
+  /*
+   * set a new href value.
+   */
+  element.firstChild.attributes.href.value = newValue;
+  return element;
+}
+
+function fixAnchor (element, newValue)
+{
+  /*
+   * replace the anchor that sits just before the heading in the DOM.
+   */
+  element.previousSibling.id = newValue;
+  return element;
+}
+
+function prettyDate (heading, dateRegex)
+{
+  /* Change ISO-like date to human-readable date. */
+  var date_options = {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric'
+  };
+  return replaceSubString(heading, dateRegex, getDate(heading, dateRegex).toLocaleDateString('en-US', date_options));
+}
+
+function getDate (string, dateRegex)
+{
+  /* Get date from string and return a date object. */
+  return new Date(string.match(dateRegex).toString());
+}
+
+function replaceSubString (originalString, dateRegex, substitution)
+{
+  /* Replace match with a new substring. */
+  return originalString.replace(dateRegex, substitution);
+}
+
+for (var item in headings)
+{
+  if (headings.hasOwnProperty(item) && headings[item].firstChild.attributes)
```

Comment:
Would it be worth creating a `var headingElm = headings[item];` variable to make the code below a bit cleaner?

Reply:
Done. See: https://github.com/tinymce/tinymce-docs/pull/1925/commits/646229562f47dd6e9cf16ae51f92abe0a5360215